requirements/common.txt

```diff
@@ -12,7 +12,7 @@ drfdocs==0.0.11
 pika==0.10.0
 pickleshare==0.7.4
 Pillow==3.4.2
-psycopg2==2.6.2
+psycopg2==2.7.3.1
```

Comment:
Please remove this change as it is not the part of this issue. Moreover, can you please tell me why you are updating the version ?

Reply:
i faced this issue while installing, 
https://github.com/psycopg/psycopg2/issues/594
After upgrading my postgres version and updating my requirements/common.txt  file made it work for me.